tests/utils_test.py

```diff
@@ -231,14 +239,19 @@ def load_and_clean_xml(location):
     for comparison. The file content is cleaned from variable parts such as
     dates, generated UUIDs and versions
     """
-    with io.open(location, encoding='utf-8') as l:
-        content = l.read()
-    data = xmltodict.parse(content, encoding='utf-8')
 
-    if 'creationInfo' in data['SpdxDocument']['Document']:
-        del(data['SpdxDocument']['Document']['creationInfo'])
+    parser = xmlparser.Parser(Builder(), StandardLogger())
+    with io.open(location, encoding='utf-8') as f:
+        document, _ = parser.parse(f)
 
-    return sort_nested(data)
+    data = {'SpdxDocument': {'Document':  TestParserUtils.to_dict(document)}}
+
+    if 'created' in data['SpdxDocument']['Document']:
+        del(data['SpdxDocument']['Document']['created'])
+    if 'creators' in data['SpdxDocument']['Document']:
+        del(data['SpdxDocument']['Document']['creators'])
+
+    return data
```

Comment:
Is `sort_nested` not necessary anymore, or why was it omitted?

Reply:
the correctness comparison is now done by comparing two json documents.